gradle/libs.versions.toml

```diff
@@ -7,7 +7,7 @@ blaze = "1.6.4"
 caffeine = "2.9.3"
 compile-testing = "0.19"
 geb = "3.4.1"
-hibernate = "5.5.9.Final"
+hibernate = ""
```

Comment:
Should this be "5.6.8.Final" as:
 https://github.com/micronaut-projects/micronaut-sql/blob/master/gradle/libs.versions.toml#L4 ?

If Micronaut SQL is publishing a BOM ? Why do we need to ahve the hibernate version here?